extract_endpoint/tests/test_endpoint.py

```diff
@@ -68,6 +68,51 @@ def upload_timestamp_file(azure_emulator_coords: azure_utils.StorageCoordinates,
     azure_service.delete_blob(azure_emulator_coords.container, endpoint.TIMESTAMP_FILENAME)
 
 
+@pytest.fixture
+def mocked_tl_app(monkeypatch, sample_secret_key: str):
+    def mock_send_to_trigger(data: typing.Dict[str, str]) -> int:
+        if 'salt' not in data:
+            return HTTPStatus.BAD_REQUEST
+        if 'signature' not in data:
+            return HTTPStatus.BAD_REQUEST
+        hasher = hashlib.new('sha3_256')
+        hasher.update((data['salt'] + sample_secret_key).encode())
+        actual_signature = hasher.hexdigest()
+        if data['signature'] != actual_signature:
+            return HTTPStatus.BAD_REQUEST
+        return HTTPStatus.CREATED
+    monkeypatch.setattr(endpoint, 'send_to_trigger', mock_send_to_trigger)
```

Comment:
mock the `send_to_trigger` function: instead of doing a `requests.post`, just fake it and return what the real post would return.